pkg/directory/directory_unix.go

```diff
@@ -1,4 +1,4 @@
-// +build linux
+// +build linux freebsd
```

Comment:
yes, _unix files need tags. We have that convention to differentiate what works on "unix-ish" systems and what works on windows.